frontend/src/assets/css/variables.scss

```diff
@@ -0,0 +1,7 @@
+:root {
+  --navbar-bg: #f8f9fa;
+  --navbar-shadow: 0 2px 4px #0000001a;
+  --navbar-hover-bg: #e2e3e5;
+  --hover-color: #000000;
+  --link-text-color: #7c7f89;
```

Comment:
The same colors can be used in different places. Do not include navbar, link in the name.